it/api/endpoints/lossClaim/list/v3/ListLossClaimsControllerISpec.scala

```diff
@@ -167,7 +186,22 @@ class ListLossClaimsControllerISpec extends V3IntegrationBaseSpec {
         response.header("Content-Type") shouldBe Some("application/json")
       }
 
-      "querying for a specific typeOfLoss" in new Test {
+      "query for everything for a TYS tax year" in new TysIfsTest {
```

Comment:
I think we also want a test where we get a successful response for the scenario where we do not provide a tax year QP for the non-TYS endpoint (as this is allowed)